.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
     - name: Install setuptools and wheel
       run: python3 -m pip install setuptools wheel
     - name: Build Docker image
-      run: bash make-distribution.sh
+      run: MAVEN_OPTS="-Xss2M" bash make-distribution.sh
```

Comment:
Keep encountering StackOverflow exception, so followed this:
https://www.lagomframework.com/documentation/1.6.x/java/JVMMemoryOnDev.html